layouts/partials/partner-form/form.html

```diff
@@ -166,11 +166,11 @@ <h2>Want to learn more about becoming a Tardigrade.io partner?</h2>
             <div class="error invalid-feedback">Please select one of the options.</div>
           </div>
 
-          <div id="step-buttons">
+          <form id="step-buttons">
```

Comment:
Form tag should start before all of the inputs and is missing action and method.